Automattic/wp-calypso
# Hosting Configuration: Update copy and icon in revamped

#### Proposed Changes

Implementing copy changes discussed here pet6gk-1j-p2

* Updating copy
* Change icon for database access

<img width="1067" alt="CleanShot 2023-01-12 at 11 48 50@2x" src="https://user-images.githubusercontent.com/1500769/211934756-f93e0e2c-b707-4939-a2ff-b56b6757f111.png">

Change depends on https://github.com/Automattic/wp-calypso/pull/71750

#### Testing Instructions



* Go to experiment page 20995-explat-experiment
* Add the `treatment` bookmarklet to your bookmark bar
* Click bookmarklet and add your user to the `treatment` cohort (tutorial video p1673249013170409/1673241333.083479-slack-C04H4NY6STW)
* `calypso.localhost:3000/hosting-config` in a site with no business plan
* You should see the updated copy

#### Pre-merge Checklist





Related to #71832
